src/app/components/FrostedGlassPromo/index.test.tsx

```diff
@@ -53,6 +53,7 @@ const Component = ({
 };
 
 describe('Frosted Glass Promo', () => {
+  // this snapshot changes and I'm not sure why
   it('when given props directly', () => {
```

Comment:
I believe it's ok it's updated since this would be the default Image behaviour now. It looks like we pass approproate Image props in for this component.